Tests/StreamChatUITests/StreamChatUITestPlan.xctestplan

```diff
@@ -0,0 +1,37 @@
+{
+  "configurations": [
+    {
+      "id": "FFB61C0B-5B74-421B-BFB6-C1FC63FEF62F",
+      "name": "Configuration 1",
+      "options": {
+      }
+    }
+  ],
+  "defaultOptions": {
+    "codeCoverage": {
+      "targets": [
+        {
+          "containerPath": "container:StreamChat.xcodeproj",
+          "identifier": "790881FC25432B7200896F03",
+          "name": "StreamChatUI"
+        }
+      ]
+    },
+    "environmentVariableEntries": [
+      {
+        "key": "CI",
+        "value": "TRUE"
+      }
+    ]
```

Comment:
This value should be set by Fastlane, it shouldn't be committed. When you run it locally, you don't want this value to be set.